src/controllers/post.js

```diff
@@ -1,15 +1,30 @@
 import { getAllPostsDb, getPostByIdDb } from '../domain/post.js'
 import { sendDataResponse } from '../utils/responses.js'
 import ERR from '../utils/errors.js'
+import dbClient from '../utils/dbClient.js'
 
 export const create = async (req, res) => {
   const { content } = req.body
+  const { id } = req.user
 
   if (!content) {
     return sendDataResponse(res, 400, { error: ERR.MISSING_CONTENT })
   }
 
-  return sendDataResponse(res, 201, { post: { id: 1, content } })
+  if (!id) {
```

Comment:
this id is coming from `req.user`. This if statement should never happen

Reply:
Lewis is correct here. If there's no user on the request object, the authentication has either failed or it's an unauthenticated request so this if check is redundant